src/routes/about.rs

```diff
@@ -17,25 +17,26 @@ use rocket::response::content;
  *
  * ## Path
  *
- * `/about`
+ * `/<chain_id>/about`
  *
  * ## Query parameters
  *
  * There are no query parameters for this endpoint
  */
-#[get("/about")]
-pub async fn info(context: Context<'_>) -> ApiResult<content::Json<String>> {
+#[get("/<chain_id>/about")]
+pub async fn get_about(context: Context<'_>, chain_id: String) -> ApiResult<content::Json<String>> {
     CacheResponse::new(context.uri())
         .duration(about_cache_duration())
-        .resp_generator(about::get_about)
+        .resp_generator(|| about::about(&context, &chain_id))
         .execute(context.cache())
         .await
 }
 
 #[doc(hidden)]
-#[get("/about/backbone")]
-pub async fn backbone(context: Context<'_>) -> ApiResult<content::Json<String>> {
-    let url = format!("{}/v1/about/", base_transaction_service_url());
+#[get("/<chain_id>/about/backbone")]
+pub async fn backbone(context: Context<'_>, chain_id: String) -> ApiResult<content::Json<String>> {
```

Comment:
nitpick: here we can have a string slice right? `chain_id: &str` and then we don't need to do `chain_id.as_str()`

Reply:
Just to be more regular with the types but I think your are right, here we could perfectly just use `&str` . I think it would actually make sense to refactor this as it is more idiomatic. We should only use `String` if we make it mutable. Otherwise we don't really add any value.